lib/ecto_job/job_queue.ex

```diff
@@ -224,11 +224,11 @@ defmodule EctoJob.JobQueue do
   The batch size is determined by `demand`.
   returns `{count, updated_jobs}` tuple.
   """
-  @spec reserve_available_jobs(repo, schema, integer, DateTime.t()) :: {integer, [job]}
-  def reserve_available_jobs(repo, schema, demand, now = %DateTime{}) do
+  @spec reserve_available_jobs(repo, schema, integer, DateTime.t(), integer) :: {integer, [job]}
+  def reserve_available_jobs(repo, schema, demand, now = %DateTime{}, base_expiry \\ 300) do
```

Comment:
Since the 300s is defaulted in `Config` I think it is fine to remove the `\\ 300` here.

Reply:
As above, would be a breaking change. Would be fine to change it if you think that's ok in the given circumstances